docs/source/api/axios-resources.md

```diff
@@ -249,6 +249,175 @@ Returns organizations belonging to the logged in user.
   }
 ```
 
+##### `postGet(data, config)`
+
+```javascript
+async postGet(data, config) {
```

Comment:
gotta update this method signature to match `addtionalPostGetArgs` as third argument to `postGet`